pyproject.toml

```diff
@@ -15,7 +15,7 @@ Flask = "^2.3.2"
 authutils = "<7"
 boto3 = "^1.17.48"
 cryptography = ">=42.0.5"
-requests = "^2.32.0"
+requests = "^2.32.2"
```

Comment:
"^2.32.0" already includes "^2.32.2", and relocking effectively upgraded `requests` to "2.32.3", so the PR description/title aren't really accurate - and if you check the lock file diff, you'll see that you upgraded many other deps too by relocking. So just "upgrading dependencies" would be fine